src/App/ApplicationManager.py

```diff
@@ -250,7 +250,27 @@ def manage_minimize(self, value=1, REQUEST=None):
         self._getDB().cacheMinimize()
 
         if REQUEST is not None:
-            REQUEST.RESPONSE.redirect(REQUEST['URL1'] + '/manage_main')
+            msg = 'ZODB in-memory caches minimized.'
+            url = '%s/manage_main?manage_tabs_message=%s' % (REQUEST['URL1'],
+                                                             msg)
+            REQUEST.RESPONSE.redirect(url)
+
+    @requestmethod('POST')
+    def manage_pack(self, days=0, REQUEST=None, _when=None):
```

Comment:
The new form / button in the ZMI is the only caller of **manage_pack** and there is no `_when` input field - so `_when` could be removed from both the signature of the method and the body.